### ISSUE TITLE
No longer supported on FF 57.0 OSX

### ISSUE THREAD

**Issue by framejk** [gmail-manager-community/gmail-manager-community#58]
FF updated today to 57.0 and this add-on is no longer supported on OSX.

**Comment by FlyingSufi** [comment]
Indeed. It seems the module system has been rethought. This extension is not compatible anymore...
A real pity, as it's my favorite as in "I use it constantly and can't get how to live without"
;)
Wish I could code well enough and solve this.

**Comment by smestarz** [comment]
Seems an issue with upgraded firefox

**Comment by smestarz** [comment]
So this wont be working in the future?
